pkgs/by-name/py/pyflyby/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+let
+  version = "1.9.6";
+in
+python3.pkgs.buildPythonApplication rec {
+  inherit version;
+  pname = "pyflyby";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "deshaw";
+    repo = "pyflyby";
+    rev = version;
+    hash = "sha256-QkoFr9tFtZ+ZEWlxe9csrzoYFl9/V2l4hKYfUWsXUdc=";
+  };
+
+  build-system = with python3.pkgs; [
+    setuptools
+    wheel
+  ];
+
+  dependencies = with python3.pkgs; [
+    six
+    toml
+    isort
+    black
+    ipython
+  ];
+
+  pythonImportsCheck = [ "pyflyby" ];
+
+  meta = {
+    description = "A set of productivity tools for Python";
+    homepage = "https://github.com/deshaw/pyflyby";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ jfvillablanca ];
+    mainProgram = "tidy-imports";
```

Uh, pretty sure `py` was the correct value for `mainProgram`.
```suggestion
    mainProgram = "py";
```

I'm not sure how many packages would conflict with that binary name, but Python doesn't so IMO it's fine to not rename it.